### PR TITLE
Update LICENSE to Apache-2.0

### PR DESCRIPTION
**Pull request by fruch**

### CODE REVIEW

**Comment by bentsi** [comment]
@fruch do we have automatic process that merges next to master?

**Comment by bentsi** [comment]
We can merge this directly to master

File: LICENSE [hunk 1]
```diff
@@ -1,661 +1,202 @@
...
+      on Your own behalf and on Your sole responsibility, not on behalf
+      of any other Contributor, and only if You agree to indemnify,
+      defend, and hold each Contributor harmless for any liability
+      incurred by, or claims asserted against, such Contributor by reason
+      of your accepting any such warranty or additional liability.
+
+   END OF TERMS AND CONDITIONS
+
+   APPENDIX: How to apply the Apache License to your work.
+
```
Comment (avikivity) [inline LICENSE]: Did you read this? these are instructions on how to use the license. You just changed the instructions below.

Comment (fruch) [reply LICENSE]: yep are correct, I'll fix this

Comment (fruch) [reply LICENSE]: fixed

**Comment by bentsi** [comment]
@avikivity please approve

**Pull request merged**